source/distributed/tria.cc

```diff
@@ -32,6 +33,7 @@
 #include <algorithm>
 #include <fstream>
 #include <iostream>
+#include <limits>
```

Comment:
```suggestion
```
I don't see where you need this.

Reply:
True, I used it but then changed my mind. Removed.